test/jasmine/tests/click_test.js

```diff
@@ -771,7 +771,7 @@ describe('Test click interactions:', function() {
             var translate = Drawing.getTranslate(mockEl),
                 scale = Drawing.getScale(mockEl);
 
-            expect([translate.x, translate.y]).toBeCloseToArray([61.070, 97.712]);
+            expect([translate.x, translate.y]).toBeCloseToArray([-25.941, 43.911]);
```

Comment:
This test was actually incorrect, as wheel events weren't passing along the mouse coordinates before https://github.com/plotly/plotly.js/pull/1522/commits/86e0d5a400766f7b2dedec98275cf619422ad869#diff-d48932d4d7f81cc4201054d397d53f1aR17 and these matter to wheel zoom.
(the change from -1000 to -20 above is irrelevant, this event handler clips at +/-20 per event fired)